Makefile

```diff
@@ -15,7 +15,7 @@
 # limitations under the License.
 #
 
-VERSION ?= latest
+VERSION ?= 0.1.0
```

Comment:
I think you said `I prefer 0.2.0-snapshot`?

Reply:
Or keep latest?

Reply:
agree